src/components/table/table.interface.ts

```diff
@@ -1,3 +1,10 @@
+export interface Item {
+  selected?: boolean;
+  // TODO: Try applying proper types to this
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
```

Comment:
In `AppPaginatedTable.vue` you have `generic="I extends Item"` so item should be extended and that `[key: string]: any;` is not necessary?

Reply:
It's required at the moment to generically access properties on the item in `AppTable`:
https://github.com/beabee-communityrm/beabee-frontend/blob/cfd0564de8ca24b7a826e73e1dc103d7787cb4e6/src/components/table/AppTable.vue#L67

It would be really amazing to make this all type safe at some point but it's way above my type skills so would take me too long at the moment!

Reply:
Okay